dataland-frontend/src/components/forms/CreateEuTaxonomyForFinancials.vue

```diff
@@ -554,14 +580,37 @@ export default defineComponent({
       try {
         this.postEuTaxonomyDataForFinancialsProcessed = false;
         this.messageCount++;
+        
+        // JSON.parse/stringify used to clone the formInputsModel in order to stop Proxy refreneces
+        const clonedFormInputsModel = JSON.parse(JSON.stringify(this.formInputsModel)) as ObjectType;
+        const kpiSectionsModel = (clonedFormInputsModel.data as ObjectType).kpiSectionsModel;
+
+        const eligibilityKpis = Object.keys(kpiSectionsModel)
+          .map( key => ({ key, fieldName: euTaxonomyKPIsModel.kpisFieldNameToFinancialServiceType[key] as string }) )
+          .map( kpi => ({ [kpi.fieldName]: kpiSectionsModel[kpi.key][kpi.fieldName] }) )
+          .reduce( (all, one) => ({ ...all, ...one }));
+
+        const kpis = Object.keys(kpiSectionsModel)
+          .filter(key => key !== 'assetManagementKpis')
+          .map( key => ({ [key]: kpiSectionsModel[key] }))
+          .map(kpi => {
+            const kpiKey = Object.keys(kpi)[0];
+            const fieldName = euTaxonomyKPIsModel.kpisFieldNameToFinancialServiceType[kpiKey];
```

Comment:
it should not be named field name but rather financialServiceType. Same for line 589